R/benchTradeStats.R

```diff
@@ -222,18 +236,23 @@ benchTradeStats <- function(Portfolio,
   if (benchmark == 'RPM') stop("RPM benchmark cannot be tested. Choose another benchmark.")
   if (length(Portfolio) < 20) warning("Statistical significance may not be be guaranteed. A 'Portfolio' with a minimum length of 20 is recommended.")
   
+  if (missing(benchmark)) benchmark <- "TradeBench" # (benchmark = 'TradeBench' & metric = -1) == Arrival Cost 
+  if (missing(metric)) metric <- (-1) # cost metric
   if (missing(side)) side <- 1
   if (missing(conf.level)) conf.level <- 0.95
   if (missing(alternative)) alternative <- "two.sided" # common naming in stats package
   
   portNames <- Portfolio
-  benchTestOut <- benchout <- list()
+  metricFullName <- ifelse(metric == 1, 'Performance', 'Cost') # consistency with benchTradePerf() naming style
+  metricAbbrName <- ifelse(metric == 1, 'Perf', 'Cost')
+  
+  benchTestOut <- benchoutstore <- list()
```

Comment:
`benchoutstore` replaces `benchout` because of a new mechanism introduced in the code. I forgot to reassign this variable on line 393, in return values. This is accounted for in next commit 161e3f59372077e8e96b872a55e35ec8b7bf5eaf, to reproduce this version please change it manually.